client/src/app/core/core-services/active-meeting-id.service.ts

```diff
@@ -51,7 +51,6 @@ export class ActiveMeetingIdService {
     }
 
     public start(): void {
-        this._meetingId = 1;
```

Comment:
re-entering this line allows my client to load fine. removing this line breaks the client

Reply:
I've let this as it was before.